p4-16/psa/PSA.mdk

```diff
@@ -433,30 +445,23 @@ assist P4 developers in debugging their code.
             // ostd.clone_class_of_service value.
         }
         if (platform_port_valid(ostd.clone_port)) {
-            create a copy of the packet and send it to the packet
-                buffer with class of service specified by
-                ostd.clone_class_of_service, after which it will start
-                egress processing with egress_port=ostd.clone_port (which
-                might be PORT_CPU to send the clone to the control CPU).
-            The contents of the packet will be the same as the packet
-                as it began ingress parsing, prepended with any headers
-                created by clone.emit() calls in the ingress deparser
-                apply block.
+            create a clone of the packet and send it to the packet
+                buffer with class of service ostd.clone_class_of_service,
+                after which it will start egress processing.
```

Comment:
The details I removed here are stated in the new 'initial values at beginning of ingress' section.  I think they are easier to find and refer to there than in this pseudocode.